app/controllers/nonsipp/membersurrenderedbenefits/SurrenderedBenefitsMemberListController.scala

```diff
@@ -100,19 +102,15 @@ class SurrenderedBenefitsMemberListController @Inject()(
               for {
                 updatedUserAnswers <- buildUserAnswersBySelection(srn, value, memberListSize)
                 _ <- saveService.save(updatedUserAnswers)
-                //TODO: update once Surrendered Benefits journey is complete/as part of transformation work
-                submissionResult <- Future.successful(Some(()))
-//              submissionResult <- if (value)
-//              {
-//                psrSubmissionService.submitPsrDetails(srn)(
-//                  implicitly,
-//                  implicitly,
-//                  request = DataRequest(request.request, updatedUserAnswers)
-//                )
-//              }
-//              else {
-//                Future.successful(Some(()))
-//              }
+                submissionResult <- if (value) {
```

Comment:
I think @ccatosdevelopment and @narendrapaduchuri you guys changed the same are, it might be worthwhile just to have a chat between yourselves and double check that you are not breaking each others' changes